File: task-tracker/src/main/java/ru/powergleb/task/tracker/frontend/tasklogic/model/Task.java
package ru.powergleb.task.tracker.frontend.tasklogic.model;


import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import ru.powergleb.task.tracker.frontend.security.model.User;

import javax.persistence.*;
import java.io.Serializable;
import java.time.LocalDateTime;

@Entity
@Table(name="tasks")
@Getter
@Setter
@NoArgsConstructor
public class Task implements Serializable {

    @Id
    @Column(name = "id")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    @Column(name = "header")
    private String header;

    @Column(name = "description", length = 4096)
    private String description;

    @Column(name = "status_active")
    private boolean statusActive;

    @Column(name = "modified")
    private LocalDateTime modified;

    @ManyToOne
//    @JsonIgnore
    @JoinColumn(name = "user_id", referencedColumnName = "id")
    private User owner;

    @Override
    public String toString() {
        return "Task{" +
                "id=" + id +
                ", header='" + header + '\'' +
                ", description='" + description + '\'' +
                ", statusActive=" + statusActive +
                ", " +
                '}';
    }
}

File: task-tracker-scheduler/src/main/kotlin/ru/powergleb/task/tracker/scheduler/model/Task.kt
package ru.powergleb.task.tracker.scheduler.model

import com.fasterxml.jackson.annotation.JsonIgnore


import java.io.Serializable
import java.time.LocalDateTime
import javax.persistence.*

@Entity
@Table(name = "tasks")
class Task(
        @Column(name = "header")
        val header: String,

        @Column(name = "description", length = 4096)
        val description: String,

        @Column(name = "status_active")
        var statusActive: Boolean,

        @Column(name = "modified")
        var modified: LocalDateTime,

        @ManyToOne
        @JsonIgnore
        @JoinColumn(name = "user_id", referencedColumnName = "id")
        val owner: User
) : BaseEntity<Int>(), Serializable